[2022-01-24 20:01:15,132] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.create_venv manual__2022-01-24T20:01:13.750109+00:00 [queued]>
[2022-01-24 20:01:15,148] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_data.create_venv manual__2022-01-24T20:01:13.750109+00:00 [queued]>
[2022-01-24 20:01:15,148] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-24 20:01:15,149] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-24 20:01:15,149] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-24 20:01:15,163] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): create_venv> on 2022-01-24 20:01:13.750109+00:00
[2022-01-24 20:01:15,173] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_data', 'create_venv', 'manual__2022-01-24T20:01:13.750109+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag_01.py', '--cfg-path', '/tmp/tmpou5eeo5_', '--error-file', '/tmp/tmp4n6btocp']
[2022-01-24 20:01:15,173] {standard_task_runner.py:77} INFO - Job 9: Subtask create_venv
[2022-01-24 20:01:15,168] {standard_task_runner.py:52} INFO - Started process 420 to run task
[2022-01-24 20:01:15,224] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_data.create_venv manual__2022-01-24T20:01:13.750109+00:00 [running]> on host 1aa77c892e3a
[2022-01-24 20:01:15,285] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=create_venv
AIRFLOW_CTX_EXECUTION_DATE=2022-01-24T20:01:13.750109+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-24T20:01:13.750109+00:00
[2022-01-24 20:01:15,286] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-24 20:01:15,287] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'virtualenv venv']
[2022-01-24 20:01:15,296] {subprocess.py:85} INFO - Output:
[2022-01-24 20:01:16,284] {subprocess.py:89} INFO - created virtual environment CPython3.6.15.final.0-64 in 378ms
[2022-01-24 20:01:16,285] {subprocess.py:89} INFO -   creator CPython3Posix(dest=/tmp/***tmpltjcs8oe/venv, clear=False, no_vcs_ignore=False, global=False)
[2022-01-24 20:01:16,285] {subprocess.py:89} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2022-01-24 20:01:16,285] {subprocess.py:89} INFO -     added seed packages: pip==21.3.1, setuptools==58.3.0, wheel==0.37.0
[2022-01-24 20:01:16,285] {subprocess.py:89} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2022-01-24 20:01:16,326] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-24 20:01:16,399] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=create_venv, execution_date=20220124T200113, start_date=20220124T200115, end_date=20220124T200116
[2022-01-24 20:01:16,433] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-24 20:01:16,503] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
